src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown

```diff
@@ -151,3 +151,34 @@ SAP Concur services will send the following HTTP headers in every request. Pleas
 `Accept`|`string`|-| SAP Concur will always set the `Accept` header to `application/json`.|
 `concur-correlationid`|`string`|-| In order to assist with troubleshooting, SAP Concur provides a unique `correlationId` in the request header. This unique code can be used during troubleshooting as it identifies the API call in the log files.|
 `Accept-Language`|`string`|-| Value of this header will follow language tag of the format LanguageCode-CountryCode, e.g. `en-US`, `fr-CA`, `de-CH` etc.|
+
+## Confirmation Codes <a name="confirmation-codes"></a>
+
+Please note current definitions of `ConfirmationCodeType` differ from what was used in [Hotel Service v2](./hotel-service-2/Introduction.html) (HSv2).
+* `RESERVATION`: Maps to OTA Unique ID Type `14` and has no equivalent for HSv2. This will be the record locator of supplier PNR and will be provided in all subsequent read/modify/cancel request on a booking. This is optional and will be mostly pass through for SAP Concur systems.
+* `SUPPLIER_CONFIRMATION`: **(Required)** Maps to OTA Unique ID Type `40` and represents supplier's confirmation number. This also be provided in all subsequent read/modify/cancel request on a booking, and it is also used in the passive segment created in the GDS. (Equivalent to `14` in HSv2).
```

Comment:
Do we need to communicate about field length constraints for this confirmation code?

Reply:
Let's see what constraints we want to add first. Only record locator I believe will be restricted to 6 (but can go up-to) 8 chars.